src/pyobs/icartt.py

```diff
@@ -17,13 +18,15 @@
 class ICARTT(object):
     """Reads ICARTT text files into Numpy arrays"""
 
-    def __init__ (self,Filenames,Alias=None,FixUnits=True,Verbose=False):
+    def __init__ (self,Filenames,Alias=None,FixUnits=True,Verbose=False,only_good=False):
```

Comment:
A suggestion - rename the 'only_good' keyword to 'strict' or perhaps the more descriptive  'ignore_geolocation'.